components/omega/src/ocn/AuxiliaryState.cpp

```diff
@@ -195,8 +234,10 @@ int AuxiliaryState::readConfigOptions(Config *OmegaConfig) {
 
    if (FluxThickTypeStr == "Center") {
       this->LayerThicknessAux.FluxThickEdgeChoice = Center;
+      this->TracerAux.TracersOnEdgeChoice         = Center;
    } else if (FluxThickTypeStr == "Upwind") {
       this->LayerThicknessAux.FluxThickEdgeChoice = Upwind;
+      this->TracerAux.TracersOnEdgeChoice         = Upwind;
```

Comment:
Do we want to be able to choose those independently ?

Reply:
This was one thing I wasn't sure about, but I suppose I might as well separate these options.